src/main/resources/assets/react4xp/executor.ts

```diff
@@ -0,0 +1,45 @@
+const inlineJsonElements = Array.from(document.querySelectorAll('script[data-react4xp-ref][type="application/json"]'));
+//console.debug('inlineJsonElements', inlineJsonElements);
+for (let index = 0; index < inlineJsonElements.length; index++) {
+	const inlineJsonElement = inlineJsonElements[index];
+	if (inlineJsonElement instanceof HTMLElement) {
+		const id = inlineJsonElement.dataset.react4xpRef;
+		//console.debug('id', id);
+
+		const json = inlineJsonElement.textContent;
+		//console.debug('json', json);
+
+		let data = {};
+		try {
+			data = JSON.parse(json);
+		} catch (e) {
+			console.error('Something went wrong while trying to JSON.parse(' + json + ')');
```

Comment:
Micro-nitpick: I guess this could be easier to read and debug if `json` was output on a new line instead of instead of inside a paranthesis.

Reply:
No one will hopefully ever see that one anyway